lib/compiler.js

```diff
@@ -38,13 +42,19 @@ exports.compile = function (code, options) {
 
   const magicString = importExportVisitor.magicString;
 
-  assignmentVisitor.visit(rootPath, {
-    exportedLocalNames: importExportVisitor.exportedLocalNames,
-    magicString: magicString,
-    modifyAST: importExportVisitor.modifyAST
-  });
+  if (importExportVisitor.madeChanges) {
```

Comment:
I think this usage lends a bit more support to the `madeChanges` terminology, yeah?